### PR TITLE
Update the Jedis/okhttp example for the latest versions of splunk-otel-java

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md [hunk 1]
```diff
@@ -6,31 +6,32 @@ Instrumentation](https://docs.oracle.com/javase/8/docs/api/java/lang/instrument/
 interface and enables tracing of many popular libraries and frameworks without
 modification of the target application's source code.
 
+In order to run this example, you will need to run the opentelemetry collector locally and have it configured to 
+receive otlp spans and export data to Splunk's ingest endpoints. 
+
+TODO: provide a link to how to do set up the collector for this demo. Or, better yet, make it more self-contained.
```
Comment (mateuszrzeszutek) [inline opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md:12]: 👍  💯 

We should use the [splunk collector distro](https://github.com/signalfx/splunk-otel-collector) if we ever make it self-contained.

**Review by mateuszrzeszutek — APPROVED** [review]

File: opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md (hunk 1 above)
Comment (jkwatson) [reply opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md:12]: Should we create an issue for this? How should it be done? docker-compose? something else?

Comment (jkwatson) [reply opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/README.md:12]: Created https://github.com/signalfx/tracing-examples/issues/67 to track it

File: opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/src/main/java/com/splunk/tracing/otel/examples/javaagent/App.java [hunk 2]
```diff
@@ -75,16 +77,13 @@ private void doFetchAndSet(String url) {
...
                 return;
             }
-
             executor.submit(() -> redisClient.set(url, respBody));
         } catch (Throwable e) {
             span.recordException(e);
-            System.out.println(e);
-        } finally {
-            span.end();
+            span.setStatus(StatusCode.ERROR, e.getMessage());
```
Comment (iNikem) [inline opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/src/main/java/com/splunk/tracing/otel/examples/javaagent/App.java:83]: I kinda expected that `recordException` already does that?

**Review by iNikem — APPROVED** [review]

File: opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/src/main/java/com/splunk/tracing/otel/examples/javaagent/App.java (hunk 2 above)
Comment (jkwatson) [reply opentelemetry-tracing/opentelemetry-java-tracing/okhttp-and-jedis/src/main/java/com/splunk/tracing/otel/examples/javaagent/App.java:83]: I also wanted/expected that, but lost that fight in the specs. Instead, we just record exception-related attributes and an event. I guess the idea is that exceptions might be recorded without it being an error in the actual span.

**Pull request merged**